examples/kitchensink/README.md

```diff
@@ -7,7 +7,5 @@ Lots of examples in one demo.
 Run:
 
 ```
-$ npm install
```

Comment:
excuse my nitpicking, but I would:

* Keep `npm install` here.
* Remove `npm install` from `make bundle`.
* Let `make bundle` only just take care of the build